lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -1292,24 +1208,17 @@ def _draw_offsetText(self, renderer):
 
 
     def _init_label(self, **kw):
-        # x in axes coords, y in display coords (to be updated at draw
-        # time by _update_label_positions)
-
         labelsize = kw.get("labelsize",
                            rcParams['axes.labelsize'])
-        #labelcolor = kw.get("labelcolor",
-        #                    rcParams['axes.labelcolor'])
         fontprops = font_manager.FontProperties(
             size=labelsize,
             weight=rcParams['axes.labelweight'])
-        textprops = dict(fontproperties = fontprops)
-                         #color = labelcolor)
 
         tr = self._axis_artist_helper.get_axislabel_transform(self.axes) \
              + self.offset_transform
 
         self.label = AxisLabel(0, 0, "__from_axes__",
-                               color = "auto", #rcParams['axes.labelcolor'],
```

Comment:
was likely intended as a comment? (unclear)

Reply:
In context, `rcParams['axes.labelcolor']` was previously used as the `color` argument, and later replaced by `"auto"` (which IIRC does fall back to the rcParam if it needs to, but may also try other default behaviour).